src/sentry/static/sentry/app/components/group/releaseStats.tsx

```diff
@@ -0,0 +1,117 @@
+import React from 'react';
+
+import LoadingIndicator from 'app/components/loadingIndicator';
+import GroupReleaseChart from 'app/components/group/releaseChart';
+import SeenInfo from 'app/components/group/seenInfo';
+import getDynamicText from 'app/utils/getDynamicText';
+import {t} from 'app/locale';
+import {Environment, Group, Organization, Project} from 'app/types';
+
+type Props = {
+  organization: Organization;
+  project: Project;
+  environments: Environment[];
+  allEnvironments: Group | undefined;
```

Comment:
maybe you could just make it optional?

Reply:
This key will always be set in this component but it might be `undefined` from the API call returning an empty object.